developer/testing/creating-test-data-meteor.md

```diff
@@ -46,3 +46,35 @@ The following factories are predefined. The appropriate object is created by exe
 
   - `variant` creates item with `type = variant`
   - `product` creates empty product without variants
+
+## Faker
+
+The [reaction-factories](https://github.com/reactioncommerce/reaction-factories) package supplies `Factory` and `faker` with additional methods for testing Reaction packages.
+
+**The recommended way to create shop:**
+
+```js
+const shopId = ReactionFaker.shops.getShop()._id;
```

Comment:
Pretty sure this ReactionFaker stuff is all gone and just comes from the Fixtures stuff

Reply:
OK, this is one article I didn't read through. I'll check it out and update.